modules/rostests/apitests/ntdll/NtSetInformationProcess.c

```diff
@@ -265,10 +266,139 @@ Test_ProcessWx86InformationClass(void)
     ok_hex(Status, STATUS_PRIVILEGE_NOT_HELD);
 }
 
+static
+void
+Test_ProcSetAlignmentProbe(void)
+{
+    NTSTATUS Status, SpecialStatus = STATUS_SUCCESS;
+    ULONG InfoClass;
+    PVOID AlignmentPointer;
+
+    /* Iterate over the process info classes and begin the tests */
+    for (InfoClass = 0; InfoClass < _countof(PsProcessInfoClass); InfoClass++)
+    {
+        /*
+         * Switch over specific information classes that return specific (special) status
+         * values when tested against Windows Server 2003. These respective status values
+         * indicate that the information class table data are correct and Windows Server 2003
+         * goes further into the code execution until NtSetInformationProcess() fails
+         * for whatever reason on these classes mentioned within the switch body.
+         */
+        switch (InfoClass)
+        {
+            case ProcessIoPortHandlers:
+            {
+                SpecialStatus = STATUS_INVALID_PARAMETER;
+                break;
+            }
+
+            /*
+             * This class returns STATUS_SUCCESS when testing
+             * for STATUS_ACCESS_VIOLATION (setting an invalid address).
+             */
+            case ProcessWorkingSetWatch:
+            {
+                SpecialStatus = STATUS_PORT_ALREADY_SET;
+                break;
+            }
+
+            case ProcessUserModeIOPL:
+            {
+                SpecialStatus = STATUS_PRIVILEGE_NOT_HELD;
+                break;
+            }
+        }
+
+        /*
+         * For classes that are not implemented in Windows Server 2003
+         * their set alignment requirement is 0 which is a valid alignment address.
```

Comment:
You cannot have an alignment requirement of 0. Smallest alignment is 1, which means no alignment requirement. Should set that (i.e. CHAR) in the table.